src/NuGet.Core/NuGet.Packaging.Core.Types/PackagingCoreConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NuGet.Packaging.Core
+{
+    public static class PackagingCoreConstants
+    {
+        public const string NupkgExtension = ".nupkg";
```

Comment:
This was just moved from another file, casing is the same. I'll make it static readonly.